src/compiler/transformers/component-native/native-static-style.ts

```diff
@@ -88,17 +89,3 @@ const createStyleLiteral = (cmp: d.ComponentCompilerMeta, style: d.StyleCompiler
 
   return ts.factory.createStringLiteral(style.styleStr);
 };
-
-const createStyleIdentifierFromUrl = (cmp: d.ComponentCompilerMeta, style: d.StyleCompiler) => {
```

Comment:
It seems to be the same function as in `src/compiler/transformers/add-static-style.ts`. Please let me know if there is any reason why the code was duplicated. Other functions in this file have also the same implementation with minor nuances. My concern here is that maybe some fixes were applied in one file but not the other.

Reply:
I think this is a safe move to make 👍 Good find!